src/ansys/motorcad/core/geometry_fitting.py

```diff
@@ -0,0 +1,213 @@
+"""Unit containing functions for drawing geometry from a list of points."""
+
+from copy import copy
+from enum import Enum
+
+from ansys.motorcad.core.geometry import Arc, Coordinate, EntityList, Line
+
+
+class _EntityType(Enum):
+    line = 0
+    arc = 1
+
+
+class _TestEntity:
```

Comment:
Used to check if list of points is within an acceptable tolerance for an entity

Reply:
can you add a docstring